docs/manifest.json

```diff
@@ -1817,6 +1817,12 @@
 		"markdown_source": "../packages/token-list/README.md",
 		"parent": "packages"
 	},
+	{
+		"title": "@wordpress/ui-utils",
```

Comment:
ui-utils as a name doesn't make sense for me. These utilities can be useful for any code. Why not `@wordpress/is`

Reply:
I guess this comment doesn't make sense if we decide to drop the package.

Reply:
> ui-utils as a name doesn't make sense for me.

The reason for this PR and for this package is to provide a space for us to migrate the other utilities found in the G2 Component system:

https://github.com/ItsJonQ/g2/tree/master/packages/utils/src

> Why not @wordpress/is

I'm happy renaming it to `@wordpress/utils` package if that were the case.

Reply:
We used to have a `@wordpress/utils` a long time ago and we removed it in favor of smaller packages with clear purpose. A `@wordpress/is` would have been a good one here but IMO, lodash offers already all of these functions and since lodash is already used across our codebase, I'd just continue using it instead of replacing it with our custom lodash.